cmd/new.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var newCmd = &cobra.Command{
 	Use:   "new",
-	Short: "New will create theme using Shopify Timber in the same directory where it gets called from",
+	Short: "New will create theme using Shopify in the same directory where it gets called from",
```

Comment:
Hi thanks for removing Timber, but this no longer makes sense. Here is a patch to fix it up.

```suggestion
	Short: "New will generate a new blank slate theme in the same directory where it gets called from and create a new theme on Shopify with those files.",
```